src/site/xdoc/server/config.xml

```diff
@@ -143,6 +143,11 @@
         <td><a href="config-blob-export.html">Blob Export Configuration</a></td>
         <td></td>
       </tr>
+      <tr>
```

Comment:
Are we missing a page for RabbitMQ here?

Reply:
it's either inside config.xml or config-guice.xml, not both. I was in config-guice.xml, I will move the blob export & blobstore configuration from config.xml to config-guice.xml